how-to/how-to-template-guide.md

```diff
@@ -0,0 +1,149 @@
+- For each step, optionally provide some background information about the task so users know what they're about to do and why. Continuing with the example, you might provide some best practices for creating memorable repository names.
+- Optionally, add a code sample or screenshot after the explanatory text, depending on the type of how-to you're writing. Screenshots are a great way to show specific parts of the screen you are referring to in a step. Make sure your code samples work.
+- Remember to orient your users when walking them through each step. If they need to open a particular file or dialogue box to complete the task, provide that information first.
+- Provide examples of sample output such as return data, and a message so that the users can validate whether they performed the step correctly or not. Continuing with the example, you might describe what happens after a user clicks the "Create repository" button.
+- Use plain language and define the terminology of any technical term next to it.
+- Include one action in a step.
+
+Here is an example step:
+
+Create repository
+
+1. Enter a name for your new repository.
+   Good repository names are short and self-explanatory. Avoid repository names with three or more words. After you click "Create repository", GitHub creates your repository and the main page for the repository is displayed. If needed, you can add substeps below a primary step. Make sure to indent the substep one tab space over if you're using Markdown.
+
+Create pull request
+
+1. 2. To create a pull request, do the following:
+
+a. Navigate to the main page of your repository.
+
+b. Under your repository name, click **Pull requests**. By default, all open pull requests are displayed.
+
+c. If you're including code samples in your steps, make sure they are also indented correctly:
+
+Set your Git username for your repository. You can change the name that is associated with your Git commits using the git config command. git config user.name "Dakota Everson"
+
+## About the "See also" section
+
+It's likely that during the course of explaining a multi-task process you will touch on other topics related to the current one, but not strictly required. This section is useful to provide your users with suggestions on further reading without interrupting the topic covered by the current document. An example might be setting up an email client, which requires working credentials to an active email address. The reader need not know how to install and run his/her own email server in order to acquire that access, although this is potentially useful. The link to documentation on running a local mail server could therefore be included in the "See also" section.
+
+## Additional resources
+
+[Diataxis framework](https://diataxis.fr/)
+
+[Docs for Developers](https://docsfordevelopers.com/)
```

Comment:
**Comment (non-blocking):** I like the resources that you've listed here, but perhaps you could explain why you recommend reading them.

Also, I'd like to recommend adding an additional resource here. I like IBM"s [Developing Quality Technical Information, 3rd ed](https://www.amazon.com/Developing-Quality-Technical-Information-Handbook/dp/0133118975/ref=sr_1_1?crid=ZJR527VZPRL6&keywords=developing+quality+technical+information&qid=1662901229&sprefix=developing+quality+t%2Caps%2C196&sr=8-1). I especially think Chapter 3, Task orientation is a great guide on how to effectively write steps. It includes many helpful examples of good and bad task writing.

The section about [lists in the Google Developer's Style Guide](https://developers.google.com/style/lists) is also really helpful. Other online style guides might be good.